### DISCUSSION TITLE
Missing data error is reported when trees converted to vcf file

### DISCUSSION THREAD

**Discussion by Liangyy1902** [tskit-dev/tskit#3216] [Q&A]
Hi,
  When I try to convert a tree file to a vcf file, I get some error message as following. How do I solve this problem?
##############Error message
**Traceback (most recent call last):
  File "/public1/home/sc60567/.conda/envs/msprime/lib/python3.8/site-packages/tskit/trees.py", line 5331, in write_vcf
    writer.write(output)
  File "/public1/home/sc60567/.conda/envs/msprime/lib/python3.8/site-packages/tskit/vcf.py", line 185, in write
    raise ValueError(
ValueError: Missing data is not currently supported. Please open an issue on GitHub if this limitation affects you.**
##############the script
**
import msprime, tskit, pyslim
ts1 = tskit.load("slim_10001.trees")
ts1 = ts1.simplify()
ts = msprime.sim_mutations(ts1, rate=6.75e-6, random_seed=1)
with open("test.vcf", "w") as vcf_file:
    ts.write_vcf(vcf_file)
**

**Comment by petrelharp** [comment]
Ah ha. Well, the solution that the message refers to is us implementing missing data output. But maybe there's a workaround - why does your data have missing data?

**Comment by jeromekelleher** [comment]
Just as a clarification here, "missing data" will be some part of the genome that has empty trees as discussed [here](https://tskit.dev/tskit/docs/stable/data-model.html#sec-data-model-missing-data)
